bodhi/server/templates/update.html

```diff
@@ -689,6 +689,15 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              <div class="p-b-1">
+                <div>
+                  <strong>CI Status</strong>
```

Comment:
Tbh, I would prefer this be a link to the actual results or somewhere where one can investigate the CI results.

However I don't think we have this URL so far, so I'm considering to add this in a second step.

Or we start with the wiki link and change it later on, also wfm.